src/koza/cli_runner.py

```diff
@@ -73,16 +71,16 @@ def transform_source(
         source_config.transform_code = str(Path(source).parent / Path(source).stem) + '.py'
 
     koza_source = Source(source_config, row_limit)
-
+    logger.debug(f"Source created: {koza_source.config.name}")
     translation_table = get_translation_table(
         global_table if global_table else source_config.global_table,
         local_table if local_table else source_config.local_table,
         logger
     )
 
-    source_koza = set_koza_app(koza_source, translation_table, output_dir, output_format, schema, logger)
-    source_koza.process_maps()
-    source_koza.process_sources()
+    koza_app = set_koza_app(koza_source, translation_table, output_dir, output_format, schema, logger)
+    koza_app.process_maps()
+    koza_app.process_sources()
```

Comment:
I would avoid making changes to code that is unrelated to the issue at hand, but no worries.. Just a thought. Makes it easier for me to review the relevant pieces!